### DISCUSSION TITLE
How to enable all posts in a page ?

### DISCUSSION THREAD

**Discussion by bloodnighttw** [adityatelange/hugo-PaperMod#224] [Q&A] [answered]
Like this: [https://adityatelange.github.io/hugo-PaperMod/fr/posts/](https://adityatelange.github.io/hugo-PaperMod/fr/posts/)
I have already searched in the whole repo but i can't find any solution.
sorry for this stupid question.

**Comment by adityatelange** [comment, marked as answer]
here https://github.com/adityatelange/hugo-PaperMod/tree/exampleSite 
is the source for https://adityatelange.github.io/hugo-PaperMod/

**Reply by bloodnighttw** [reply]
Thanks.
